packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -49,6 +50,16 @@ export class VaAccordionItem {
    */
   @Prop() level?: number = 2;
 
+  /**
+   * The id of the accordion item, required for accessibility. (USWDS only)
+   */
+  @Prop() bordered?: boolean = false;
```

Comment:
Is the documentation text correct for this prop?

```
  /**
   * The id of the accordion item, required for accessibility. (USWDS only)
   */
  @Prop() bordered?: boolean = false;
```

Reply:
This was incorrect and I updated it, I also realized that I needed to clean up the unused bordered prop on the va-accordion.tsx file. Great find!